### PR TITLE
Enable trailing slash redirects

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Moved dynamic redirects below static redirects and enable trailing slash redirects

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

File: tools/deploy-static.sh [hunk 1]
```diff
@@ -14,10 +14,10 @@ DIVISION_LINE=`awk '/####*/ {print NR}' config/redirects.txt | sed -n 3p`
 head -n $DIVISION_LINE config/redirects.txt > build/_redirects
 
 # normalize all Cloudflare Style Redirects to end with /
-# sed -r 's/^(\/[^ ]+\/[^\/ ]+) /\1\/ /g' -i build/_redirects
+sed -r 's/^(\/[^ ]+\/[^\/ ]+) /\1\/ /g' -i build/_redirects
 
 # duplicate every Cloudflare Style Redirect ending with / to also end without /
-# sed -rn 'p; s/^(\/[^ ]+)\/ /\1 /p' -i build/_redirects
+sed -rn 'p; s/^(\/[^ ]+)\/ /\1 /p' -i build/_redirects
```
Comment (klutchell) [inline tools/deploy-static.sh:20]: Don't enable these, I'm removing them in a subsequent PR.

File: config/redirects.txt [hunk 2]
```diff
@@ -127,6 +121,15 @@
...
 
 
+# Important: keep dynamic redirect below the static redirects
+# https://developers.cloudflare.com/pages/platform/redirects/
+# Dynamic redirects
+
+/deployment/network/:version/ /reference/OS/network/
+/understanding/understanding-devices/:version/ /reference/OS/overview/
+/reference/OS/overview/:version/ /reference/OS/overview/
+/reference/OS/network/:version/ /reference/OS/network/
```
Comment (klutchell) [inline config/redirects.txt:131]: These are new, I'll need to update my regex to ignore these.

**Review by klutchell — CHANGES_REQUESTED** [review]

File: config/redirects.txt (hunk 2 above)
Comment (vipulgupta2048) [reply config/redirects.txt:131]: Have always been there?

File: tools/deploy-static.sh (hunk 1 above)
Comment (vipulgupta2048) [reply tools/deploy-static.sh:20]: If that's the case you can take up these changes in your PR. I am fine with it.

Comment (vipulgupta2048) [reply tools/deploy-static.sh:20]: Can close this one

File: config/redirects.txt (hunk 2 above)
Comment (klutchell) [reply config/redirects.txt:131]: Oh, I never noticed the colons before. Do they also need both slash and non-slash variants?

Comment (vipulgupta2048) [reply config/redirects.txt:131]: Screenshot from 2023-01-11 01-18-16

We can improve the regex but works just fine ^^

**Pull request closed without merging**